CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 - Major: Newly uploaded Twitch emotes are once again present in emote picker and can be autocompleted with Tab as well. (#2992)
 - Major: Added the ability to add nicknames for users. (#137, #2981)
-- Major: Work on rate-limiting JOINs and PARTs. (#3112)
+- Major: Fixed constant disconnections with more than 20 channels by rate-limitting outgoing JOIN messages. (#3112, #3115)
```

Comment:
Decided to rephrase the changelog a little, could some kind of God of War @Felanbird have a look and make sure this is clear and descriptive enough?

Reply:
```suggestion
- Major: Fixed constant disconnections with more than 20 channels by rate-limiting outgoing JOIN messages. (#3112, #3115)
```

Reply:
@Felanbird Please also consider this: https://github.com/Chatterino/chatterino2/pull/3115#discussion_r682809758 :smile:

Reply:
Resolving in favour of https://github.com/Chatterino/chatterino2/pull/3115#discussion_r682911557